db/fixtures/lessons/advanced_html_and_css_lessons.rb

```diff
@@ -0,0 +1,118 @@
+def advanced_html_and_css_lessons
```

Comment:
I'm not sure if we should split these up, I really like what we've done with the Git and React lessons recently by having all of the lessons related to those subjects in one fixture file instead of mixed into the courses as they are displayed on the site. 

The intention of these fixtures was always to do something like that and have the course seed files be where we mixed them together as needed for the courses. Although foundations is a bit of a special case in which there are so many different kinds of lessons it was easier to include most of them in one file rather than try to classify them. 

Whats your thoughts @01zulfi?

Reply:
I understand having lessons of a related subject be in one file i.e having Git and React seed files. . This brings up the problem where classification becomes tedious and just placing them together makes sense like the foundation lessons.  I believe this is inconsistent, and we should strive for consistency. On the other hand, I'm not entirely sure about mixing lessons of different courses even if they can be classified.

Separating these files based on the course structure on the website provides that consistency. The foundation seeds won't be a special case anymore rather it would make sense. The React seed file would need to be merged into the JavaScript seed file (which I'm planning to address in a future PR if approved). Having Advanced HTML/CSS & Intermediate HTML/CSS seeds separate will add to that consistency. And above all, lessons will be easier to find. 

The only exception would be the Git lesson seeds, but the exception is justified due to its unique nature in the curriculum. 

I've shared my thoughts above, although I'm happy with leaving the final call to you, @KevinMulhern.

Reply:
Thanks for your perspective I can definitely see it from your point of view, and I can see value in having that consistency.

For me, the [course seed files](https://github.com/TheOdinProject/theodinproject/tree/main/db/fixtures/paths) are where we should have that consistency instead, they are responsible for providing the one to one mapping with what is on the site. The lesson fixture file's responsibility is to be a repository of all the available content we can use across those courses. 

The recent Git changes will become the norm rather than the exception IMO. We have been talking a lot more about interleaving lessons from different subjects into courses for a better flow.

For example, on the Rails path, I want to add a couple of JavaScript projects to the Ruby course so students can keep those skills fresh. If the lesson fixtures are one to one with the courses, I would need to add those JS projects to the ruby_lessons file. This is fine, but if we thought those projects should be included in the JS path too at some point, we would have calls to the ruby_lessons file from the JS course seed file which could get very confusing.

Reply:
Ah, that makes much more sense. Thanks for the detailed response; I'll update this PR accordingly.